components/shared/LinksInput.js

```diff
@@ -0,0 +1,198 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Grid, Cell } from 'styled-css-grid';
+
+import { GenerateUuid } from '../../utilities/utilityFunctions';
```

Comment:
I always for get that `utilities` has an index so that we can export the supporting functions into it and just import from there. If you add `validation` and `utilityFunctions` into the index you can combine these imports from `utilities` into one call.